.github/workflows/release_scripts/prisma_vulnurability_checker.py

```diff
@@ -23,7 +23,7 @@ def get_excluded_packages():
 
     exclusions = set()
     for exclusion in whitesource_exclusion_entries:
-        exclusions.add(exclusion['package'])
+        exclusions.add(exclusion['packageName'])
```

Comment:
field name has changed to `packageName` in https://us-east-1.console.aws.amazon.com/dynamodbv2/home?region=us-east-1#item-explorer?table=prisma-excluded-packages&maximize=true